docs/cli-reference.md

```diff
@@ -338,19 +338,20 @@ export HTTPS_PROXY="http://10.10.1.10:1080"
 
 <!-- Source code reference - the exit codes are located in the Semgrep repository - https://github.com/returntocorp/semgrep/blob/develop/cli/src/semgrep/error.py. -->
 
-Semgrep can finish with the following exit codes:
+Semgrep OSS can finish with the following exit codes:
```

Comment:
Should we say OSS (as these exit codes are coming from the OSS part of our software), or just Semgrep to emphasize that whatever product our users will use, these are always the exit codes they can expect? @milanwilliams

Reply:
I'd say just Semgrep! :)

Reply:
```suggestion
Semgrep can finish with the following exit codes:
```